### ISSUE TITLE
different dmx devices addresses?

### ISSUE THREAD

**Issue by moebiussurfing** [TinkerKit/DmxMaster#3]
hi, i just purchased 2 tinkerkit dmx modules.
i am just starting to test...
there is any way to use multiple devices, each  with his own address?

**Comment by moebiussurfing** [comment]
sorry, was my fault. now I understand it.

**Comment by mdreeke** [comment]
Hi moebiussurfing.  What did you find out about using the TinkerKit DMX Master Shield with multiple DMX devices (multiple device addresses)?  Is it possible?

**Comment by moebiussurfing** [comment]
yes @mdreeke , it was my fault, was very easy. not remember now, but check the examples and you will get it work easy. ie: if you have a beamer A with chanels 1-2-3-4 and another B with 1-2, you must put the start address into A to 1 and the start address 5 to the B. Then from your soft you can control from channel 1 to 4 to the A beamer and from 5 and 6 to the B beamer. hope helps
